rl_chain/pick_best_chain.py

```diff
@@ -19,6 +21,26 @@
 from sentence_transformers import SentenceTransformer
 
 
+class PickBestLabel(base.Label):
+    chosen_action: int
+    chosen_action_probability: float
+    cost: Optional[float]
+
+    def __init__(self, vwpred: List[Tuple[int, float]], cost: Optional[float] = None):
```

Comment:
Possibility to provide cost before sampling (without knowing which action is chosen) seems weird. Looks like we are not calling it that way anyways - should we maybe clean it up?

Reply:
yeah it is a bit weird because cb is the only reduction that doesn't do sampling of the pmf before it returns the result, let me think about it a bit